dataframely/columns/_base.py

```diff
@@ -79,6 +93,45 @@ def validate_dtype(self, dtype: PolarsDataType) -> bool:
 
     # ---------------------------------- VALIDATION ---------------------------------- #
 
+    def _get_rule_names(
+        self, rules: list[Callable[[pl.Expr], pl.Expr]], count_separator: str = "__"
```

Comment:
```suggestion
        self, rules: list[Callable[[pl.Expr], pl.Expr]]
```
IMO it would be OK to just commit to this being one value, since I don't think we'll every use different ones.
Should we just make this a constant `RULE_NAME_SUFFIX_SEPARATOR`?